unlock-app/src/components/lock/ShowUnlessUserHasKeyToAnyLock.js

```diff
@@ -0,0 +1,33 @@
+import { connect } from 'react-redux'
+import PropTypes from 'prop-types'
+import UnlockPropTypes from '../../propTypes'
+
+export const ShowUnlessUserHasKeyToAnyLock = ({ locks, keys, children }) => {
+  const now = new Date().getTime() / 1000
+  for(let lock of locks) {
```

Comment:
I'm wondering if this logic should be moved to a helper so it can be reused elsewhere (maybe in a future PR).

Reply:
sure!